src/main/kotlin/biokotlin/cli/MutateProteins.kt

```diff
@@ -0,0 +1,363 @@
+package biokotlin.cli
+
+import biokotlin.seq.ProteinSeqRecord
+import biokotlin.seqIO.FastaIO
+import biokotlin.seqIO.SeqType
+import biokotlin.util.bufferedReader
+import biokotlin.util.bufferedWriter
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.parameters.options.default
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import com.github.ajalt.clikt.parameters.types.boolean
+import com.github.ajalt.clikt.parameters.types.enum
+import com.github.ajalt.clikt.parameters.types.int
+import com.google.common.collect.HashMultimap
+import com.google.common.collect.Multimap
+import org.apache.logging.log4j.LogManager
+import java.io.BufferedWriter
+import kotlin.random.Random
+
```

Comment:
Needs a header comment explaining the main points, input/output, of this command

Reply:
Ok, I added this